### ISSUE TITLE
[PR fails] Adding Stricter Type Checks For Input Object Types Fails only on Node8/9

### ISSUE THREAD

**Issue by isocroft** [IBM/openapi-to-graphql#321]
With respect to my recent [PR](https://github.com/IBM/openapi-to-graphql/pull/320) to this awesome project, I had been researching ways to add stricter type checks to a code-first code-generated graphql schema. I found [this package](https://github.com/joonhocho/graphql-scalar) and included it in the **openapi-to-graphql** package to sanitize and validate input types for **OpenAPI v3.x.x** _parameters_ and _requestBody_ via their OAS schema definitions. 

All Tests Pass on Travis CI/CD (NodeJS v10/11):

![image](https://user-images.githubusercontent.com/5495952/81502651-166baa00-92d7-11ea-9bfe-0a81df23a283.png)

All Tests Pass on Local (NodeJS v10.17.0)

![image](https://user-images.githubusercontent.com/5495952/81502677-34390f00-92d7-11ea-835a-33ef84ecd700.png)

As you see above: All tests on my local copy of the project pass (PS: I am running NodeJS v10.17.0) but on the Travis CI/CD it fails specifically on Node 8/9 and passes on Node 10/11/12.

I suspect that the **MQTT subscriptions** [package](https://github.com/davidyaha/graphql-mqtt-subscriptions) is the reason why. But I wanted help in debugging the issue.

**Comment by isocroft** [comment]
Here is the list of build jobs and which tests pass and which fail:

![image](https://user-images.githubusercontent.com/5495952/81503040-5fbcf900-92d9-11ea-9f68-f23286d8b609.png)

**Comment by Alan-Cha** [comment]
@isocroft Thanks for filling this issue! I think you are correct though. I'm fairly sure this problem is originating from the MQTT subscription package. I'll take a look...

**Comment by Alan-Cha** [comment]
Node v8 and v9 are both [deprecated](https://github.com/nodejs/Release). I think in an effort to save everyone, we shouldn't look too deeply into these broken tests. I will update the Travis config to reflect this.

**Comment by isocroft** [comment]
@Alan-Cha  I looked into this some more and I found out that `graphql-mqtt-subscriptions` package has a dependency : `graphql-subscriptions@0.4.2` but then the _package.json_ specifies a different one: `graphql-subscriptions@1.1.0` and they seem to be clashing.

**Comment by Alan-Cha** [comment]
Hmmm but the tests still work with the other Node versions even though the `graphql-mqtt-subscriptions` is out of sync. Additionally, I couldn't replicate this error using the same Node version as the one used in the test (v8.14.0). We experienced similar problems when we first pulled in the subscription PR #311 (related #297) but I thought we addressed them. Either way, I still think that we should stop testing on v8, v9, and v11 and unless the LTS versions complain, I don't think we should look into these broken tests too deeply.
